crates/websocket-proxy/src/main.rs

```diff
@@ -18,6 +18,8 @@ use clap::Parser;
 use dotenvy::dotenv;
 use metrics_exporter_prometheus::PrometheusBuilder;
 use rate_limit::RedisRateLimit;
+#[cfg(feature = "compression")]
```

Comment:
@haardikk21 what do you think about using an arg for this? (can add [here](https://github.com/flashbots/rollup-boost/blob/main/crates/websocket-proxy/src/main.rs#L31)).

Feels like a bit overkill to add a new compile feature for this, I can imagine this is something we may want to toggle quickly on a devnet etc without having to recompile